.github/workflows/ci.yml

```diff
@@ -390,15 +390,19 @@ jobs:
           git fetch upstream
           if git diff upstream/master --name-only | grep -q "^asv_bench/"; then
               asv machine --yes
-              asv run --quick --show-stderr --launch-method=spawn -b ^benchmarks -b ^io -b ^scalability \
-                | sed "/failed$/ s/^/##[error]/" | tee benchmarks.log
-              # check omnisci backend
-              MODIN_BACKEND=omnisci MODIN_EXPERIMENTAL=true asv run --quick -b ^omnisci \
-                --show-stderr --launch-method=forkserver --config asv.conf.omnisci.json \
-                | sed "/failed$/ s/^/##[error]/" | tee benchmarks.log
-              if grep "failed" benchmarks.log > /dev/null ; then
```

Comment:
why `sed` and `grep` are no longer there? were they not needed?

Reply:
Previously, ASV would always return 0, even if the benchmarks failed. In this case, we used parsing of logs in order to return an error code other than 0. The new version of the program has `--strict` option that returns an error code other than 0 if benchmarks fail, which eliminates the need to parse the logs.

Reply:
Where is ASV version update then?..

Reply:
This feature already in ASV that we take from master https://github.com/modin-project/modin/blob/7727c23adc58c470b6008dc38519da7d21c3ac0b/environment-dev.yml#L42